.pre-commit-config.yaml

```diff
@@ -94,3 +94,8 @@ repos:
   rev: 0.23.2
   hooks:
     - id: check-github-workflows
+
+- repo: https://github.com/stefmolin/numpydoc
```

Comment:
We should use `numpy/numpydoc` here instead. This repo was merged into the upstream for the upcoming release.